render/detailed_node.go

```diff
@@ -64,23 +64,23 @@ func (r sortableRows) Less(i, j int) bool {
 	}
 }
 
-type tables []Table
+type sortableTables []Table
```

Comment:
It wasn't strictly necessary, I did it for consistency with the `rows` -> `sortableRows`renaming.